Sources/Fluent/Relations/RelationError.swift

```diff
@@ -3,6 +3,7 @@
 /// Ex: Children, Parent, Siblings
 public enum RelationError {
     case idRequired(Entity)
+    case oneToOneConstraint(Entity, Entity.Type, Int)
```

Comment:
Agreed. It was not an attempt to add one-to-one support to Fluent, just to check a constraint on one on the helper functions.